benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherAzureBenchmark.cs

```diff
@@ -31,7 +30,6 @@ public void Setup()
 
             _baseline = (BarebonesMatcher)SetupMatcher(new BarebonesMatcherBuilder());
             _dfa = SetupMatcher(CreateDfaMatcherBuilder());
-            _tree = SetupMatcher(new TreeRouterMatcherBuilder());
```

Comment:
Since we don't have HttpMethodEndpointConstraint we can't support this any more. I don't feel like I'm missing much by deleting it.